### ISSUE TITLE
NullPointerException BookmarkFragment in updatePendingBookmarksToDelete

### ISSUE THREAD

**Issue by Amejia481** [mozilla-mobile/fenix#20872]
Low volume.
 
Sentry :
* Nightly https://sentry.prod.mozaws.net/operations/firefox-nightly/issues/12052102/
* Beta https://sentry.prod.mozaws.net/operations/firefox-beta/issues/11800404
* Release https://sentry.prod.mozaws.net/operations/firefox/?query=is%3Aunresolved+updatePendingBookmarksToDelete

```
java.lang.NullPointerException: null
    at org.mozilla.fenix.library.bookmarks.BookmarkFragment.updatePendingBookmarksToDelete(BookmarkFragment.kt:2)
    at org.mozilla.fenix.library.bookmarks.BookmarkFragment.deleteMulti(BookmarkFragment.kt:5)
    at org.mozilla.fenix.library.bookmarks.BookmarkFragment.access$deleteMulti(BookmarkFragment.kt:1)
    at org.mozilla.fenix.library.bookmarks.BookmarkFragment$onCreateView$4.invoke(BookmarkFragment.kt:3)
    at org.mozilla.fenix.library.bookmarks.DefaultBookmarkController.handleBookmarkDeletion(BookmarkController.kt:1)
    at org.mozilla.fenix.library.bookmarks.BookmarkFragmentInteractor.onDelete(BookmarkFragmentInteractor.kt:15)
    at org.mozilla.fenix.library.bookmarks.viewholders.BookmarkNodeViewHolder$1.invoke(BookmarkNodeViewHolder.kt:5)
    at org.mozilla.fenix.library.bookmarks.BookmarkItemMenu$menuItems$6.invoke(BookmarkItemMenu.kt:1)
    at mozilla.components.browser.menu2.adapter.TextMenuCandidateViewHolder.onClick(TextMenuCandidateViewHolder.kt:1)
    at android.view.View.performClick(View.java:7217)
    at android.view.View.performClickInternal(View.java:7191)
    at android.view.View.access$3500(View.java:828)
    at android.view.View$PerformClick.run(View.java:27679)
    at android.os.Handler.handleCallback(Handler.java:900)
    at android.os.Handler.dispatchMessage(Handler.java:103)
    at android.os.Looper.loop(Looper.java:219)
    at android.app.ActivityThread.main(ActivityThread.java:8349)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1055)
```

**Comment by Amejia481** [comment]
Sentry issue: [FIREFOX-NIGHTLY-BYW](https://sentry.prod.mozaws.net/operations/firefox-nightly/issues/12052102/?referrer=github_plugin)

**Comment by Amejia481** [comment]
Sentry issue: [FIREFOX-NIGHTLY-C5M](https://sentry.prod.mozaws.net/operations/firefox-nightly/issues/12273100/?referrer=github_plugin)

**Comment by gabrielesvelto** [comment]
This is spiking, we have almost 500 crashes on the release channel in the past week.

**Comment by Amejia481** [comment]
I chatted with @SoftVision-LorandJanos, he wasn't able to replicated the issue given the above STR, as the issue was spotted looking to Sentry we don't have a defined set of steps, the most solid confirmation that we have is the [Sentry reports](https://sentry.io/organizations/mozilla/issues/3169122506/?project=6295546&query=is%3Aunresolved+updatePendingBookmarksToDelete), so far we haven't had any new reports after the patch, I think it's OK to close as [a reference the AC](https://sentry.io/organizations/mozilla/issues/3169122506/events/?project=6295546) version when this patch landed was `101.0.20220420143047`
